src/helpers/uploadFiles.test.ts

```diff
@@ -21,31 +20,9 @@ describe("uploadFiles tests", () => {
     const files = [new File([], "file1"), new File([], "file2")];
 
     const result = await uploadFiles(files, bucket);
-    expect(global.fetch).toHaveBeenCalledTimes(2);
     expect(result).toStrictEqual(baseURL);
   });
 
-  test("check generated call parameters", async () => {
-    Date.now = vi.fn(() => TIME_STAMP);
-
-    const file = new File([], " test file name");
-
-    await uploadFiles([file], bucket);
-    expect(global.fetch).toHaveBeenCalledTimes(1);
-
-    expect(global.fetch).toHaveBeenCalledWith(APIs.aws + "/v2/file-upload", {
```

Comment:
asserting request is [not recommended](https://mswjs.io/docs/best-practices/avoid-request-assertions/)